scripts/db_migrator.py

```diff
@@ -485,6 +485,20 @@ def migrate_qos_fieldval_reference_format(self):
         self.migrate_qos_db_fieldval_reference_remove(qos_table_list, self.configDB, self.configDB.CONFIG_DB, '|')
         return True
 
+    def migrate_port_qos_map_global(self):
+        """
+        Generate dscp_to_tc_map for switch.
+        """
+        dscp_to_tc_map_table_names = self.configDB.get_keys('DSCP_TO_TC_MAP')
+        if len(dscp_to_tc_map_table_names) == 0:
+            return True
+        
+        qos_maps = self.configDB.get_table('PORT_QOS_MAP')
+        if 'global' not in qos_maps.keys():
```

Comment:
I suggest skipping Mellanox devices here because the global `PORT_QOS_MAP` is not required and will not be applied on Mellanox devices.
```suggestion
        if 'global' not in qos_maps.keys() and self.asic_type != "mellanox":
```

Reply:
Thanks. I added a check for ASIC type. The table entry is only added for a platform now.